spec/inverted-index-spec.js

```diff
@@ -1,74 +1,96 @@
 const InvertedIndexhelper = require('./inverted-index-helper');
-const IndexKlass = require('./inverted-index');
+const IndexClass = require('./inverted-index');
+const empty = require('../spec/empty');
+const invalidFile = require('../spec/invalid.json');
 const fs = require('fs');
 const path = require('path');
 
 const book = fs.readFileSync(path.resolve(__dirname, 'books.json'));
 
-
 describe('Inverted Index', () => {
-  const helpers = new InvertedIndexhelper();
-  const InvertedIndex = new IndexKlass(InvertedIndexhelper);
+  const InvertedIndex = new IndexClass(InvertedIndexhelper);
   const file1 = book;
-  const emptyFile = [];
-  const notJson = 'books.js';
-  InvertedIndex.files.books = JSON.parse(book);
-  InvertedIndex.createIndex('books');
+  const file2 = empty;
```

Comment:
### Critical

Why did you rename these to less meaningful variable name, it's harder to discern the logic where you used them below.